docs/old-reference-guide/modules/tuning/pages/rdbms-tuning.adoc

```diff
@@ -0,0 +1,239 @@
+This indicates this file should be used to override existing annotations, instead of replacing them.
+For the best results, ensure that the `DomainEventEntry`  table uses its own sequence.
+This can be ensured by specifying a different sequence generator for that entity only.
+
+== PostgreSQL, `@LOB`-annotated columns, and default Hibernate mapping
+
+Specific to link:https://www.postgresql.org/[PostgreSQL] is its link:https://wiki.postgresql.org/wiki/TOAST[TOAST] functionality.
+TOAST stands for "the oversized attribute storage technique," ensuring columns that are (by default) larger than 8KB are:
+
+1. Compressed **if** possible.
+If this is insufficient,
+2. the wide columns is broken into chunks and moved to another table.
+
+Any field that is moved out of line to this _other_ "large object storage table" will be replaced by an OID field.
+This "object identifier" will allow the main table to point to the actual data in the large object storage table.
+
+Although this is a very useful feature for PostgreSQL users to help with large row sizes, it has some impact too, being:
+
+1. There is some added overhead when reading data, as the original and the large object table should be read.
+2. Removing an object that has a TOAST-ed field does not automatically remove it from the large object table.
+3. Manually querying the table will result in an OID column instead of the actual data being shown.
+
+Let us look how this behavior from PostgreSQL combines with Axon Framework when combined with JPA and Hibernate.
+
+Axon Framework uses the `@Lob` in several columns and tables for its JPA-based support, being:
+
+* The `payload` and `meta_data` columns in the `domain_event_entry` table.
+* The `payload` and `meta_data` columns in the `snapshot_event_entry` table.
+* The `token` column in the `token_entry` table.
+* The `serialized_saga` column in the `saga_entry` table.
+* The `diagnostics`, `payload`, `metadata`, and `token` column in the `dead_letter_entry` table.
+
+Furthermore, Hibernate will by default use the aforementioned OID-type whenever an `@Lob` annotation is found.
+Thus, the dedicate large object storage solution will be used if you are using PostgreSQL to store events, snapshots, tokens, sagas, and dead letters.
+
+As events and snapshots are frequently read, the overhead predicament discussed earlier will be hit.
+But arguably more problematic is issue two, especially for the `token_entry` table.
+
+The "claim" on a token is very frequently updated to allow correct collaboration in a distributed Axon setup (please read our xref:axon-framework-reference:events:event-processors/streaming.adoc#tracking-tokens[Tracking Tokens] section for more details).
+As the large object table is **not** automatically cleared, it will eventually overflow through all the updates.
+
+Hence, it would be best to avoid this "enforced" TOAST behavior through Hibernate's default settings.
+Luckily, that can be done with three easy steps:
```

Comment:
It's unclear to me whether we need all 3 steps to be performed, or if changing the mapping is already enough. It looks like the first two steps are mutually exclusive. One forces the storage type for LOB to bytea through the dialect, the other explicitly configures bytea for the column in the database.